File: distros/J/JavonetPerlSdk/lib/Javonet/Binaries/Python/javonet/core/transmitter/Transmitter.py
from javonet.core.transmitter.TransmitterWrapper import TransmitterWrapper


class Transmitter:

    @staticmethod
    def send_command(message):
        return TransmitterWrapper.send_command(message)

    @staticmethod
    def activate(license_key):
        return TransmitterWrapper.activate(license_key)

    @staticmethod
    def set_config_source(source_path):
        return TransmitterWrapper.set_config_source(source_path)

File: distros/J/JavonetPerlSdk/lib/Javonet/Binaries/Nodejs/lib/core/transmitter/Transmitter.js
const wrapper = require('./TransmitterWrapper')

class Transmitter {
    static initialize() {
        return wrapper.initialize()
    }

    static sendCommand(messageArray) {
        return wrapper.sendCommand(messageArray)
    }

    static activate = function (licenseKey) {
        return wrapper.activate(licenseKey)
    }

    static setConfigSource(configSource) {
        return wrapper.setConfigSource(configSource)
    }
}

module.exports = Transmitter
